setup.cfg

```diff
@@ -1,48 +0,0 @@
-[bumpversion]
-current_version = 1.8.0
-commit = True
-tag = False
-message = Bump up to version {new_version}.
-
-[metadata]
-description-file = README.md
-
-[bdist_wheel]
-python-tag = py3
```

Comment:
Did we lose the python tag specifier?

Reply:
This is outdated. I found a few different random threads indicating that this section is no longer necessary, but the most authoritative source is the PyPA's wheel documentation and the only usage of this section I found anywhere was around [explicitly declaring that a wheel is universal](https://packaging.python.org/en/latest/guides/distributing-packages-using-setuptools/#wheels). I also tested building wheels both with and without this section prior to moving config info from setup.cfg to pyproject.toml, and then I build wheels again after the switch. In all cases the produced wheel had the same tag: `signac-1.8.0-py3-none-any.whl`.